scripts/geth-genesis.json

```diff
@@ -24,7 +28,7 @@
   "timestamp": "0x60ed6d23",
   "extraData": "0x000000000000000000000000000000000000000000000000000000000000000057f96e6b86cdefdb3d412547816a82e3e0ebf9d20000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
   "gasLimit": "0x4db9760",
-  "difficulty": "0x1",
+  "difficulty": "0x0",
```

Comment:
why changing the difficulty?

Reply:
yes, post merge this field has to be set to 0